media-libs/mlt/mlt-6.18.0.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="An open source multimedia framework"
+DESCRIPTION="It provides a toolkit for video editors and many more types of applications."
```

Comment:
Line is still over 80 chars, check the guidelines: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines